lightning/src/routing/router.rs

```diff
@@ -45,13 +45,35 @@ pub trait Router {
 #[cfg(not(any(test, feature = "_test_utils")))]
```

Comment:
Comment above needs updating since the e.g. the direction boolean is no longer publicly exposed. Maybe keep it high level like "Tracker for in-flight HTLCs, used for pathfinding. .."

Reply:
hm, good point. I think it's still useful for expressing how the underlying data is organized, so I will move it to the struct declaration as a regular code comment